src/object.c

```diff
@@ -1295,7 +1295,7 @@ size_t objectComputeSize(robj *key, robj *o, size_t sample_size, int dbid) {
             if (samples) asize += (double)elesize / samples * raxSize(s->cgroups);
         }
     } else if (o->type == OBJ_MODULE) {
-        asize = moduleGetMemUsage(key, o, sample_size, dbid);
+        asize = zmalloc_size((void *)o) + moduleGetMemUsage(key, o, sample_size, dbid);
```

Comment:
It looks like `zmalloc_size((void *)o)` is part of every case in this long else-if chain. We could factor it out and initialize `size_t asize = zmalloc_size((void *)o);` once for all cases.

Reply:
Agree. I think it was attempted to keep the existing staructure when it was modified in https://github.com/valkey-io/valkey/pull/2290, but your point is valid

Reply:
If you want to keep the change small, to make it easier to backport, etc., just merge it as is. :+1: